docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -1,280 +1,366 @@
+* <span style="color:red">⓹</span> Toggle SSL as needed.
+: For this quickstart tutorial consider **Disable**, as you may not have signed SSL Certificate in your testing OpenShift environment and the webhook will fail to deliver it's payload with the error: `We couldn’t deliver this payload: x509: certificate signed by unknown authority`.
+* <span style="color:red">⓺</span> Click **Add webhook**
+
+Now your webhook should be added and green success icon should appear left to your webhook's URL, which means it succesfully sent the first `ping` payload to your OpenShift URL endpoint:
 
-* Navigate to https://github.com/your_org/mig-demo-apps/settings/hooks
-* Click **Add webhook**
-* Paste the URL in **Payload URL**, with the real secret replacing the text `<secret>`
-* Select content type **application/json**
-* Toggle SSL as needed, for testing consider disabling
-* Click **Add webhook**
+![github_webhook_3](../img/github_webhook_3.png)
 
 For more information, consult [GitHub's documentation](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks).
 
-#### Update the application source code
+##### Update the application source code
 
-In your OpenShift mongo-persistent project page, open the link to the sample application or get the link, by running
+In your OpenShift `mongo-persistent` project page, open the link to the sample application or get the link, by running
 ```
-oc get route todolist-route --namespace mongo-persistent -o=go-template='http://{{.spec.host | printf "%s\n" }}'
+$ oc get route todolist-route --namespace mongo-persistent -o=go-template='http://{{.spec.host | printf "%s\n" }}'
 ```
 
-It should look like in the following image.
+The application should look like in the following image:
 
 ![todolist](../img/todolist_orig.png)
 
-The text "Enter an activity" does not seem clear, let's change that to "Add a todo", by running
-```
-sed -i 's/Enter an activity/Add a todo/g' index.html
-```
+The text **"Enter an activity"** does not seem clear, let's change that to **"Add a todo item"**, by running:
 
-If you are happy with the change, commit and push
+```shell
+$ sed -i 's/Enter an activity/Add a todo item/g' index.html
 ```
-git add index.html
-git commit -m "update text box"
-git push origin master
+
+If you are happy with the change, commit it and push to the default `master` branch:
+
+```shell
+$ git add index.html
+$ git commit -m "Updated index.html activity text."
+$ git push origin master
 ```
 
-Once the commit is pushed to the repository, it will automatically rebuild because we have setup the GitHub webhook.
+Once the commit is pushed to the repository, it will automatically rebuild because we have setup the [GitHub webhook](#github-webhook).
+
+You will now see that the todolist application starts to rebuild, like in the following image or by running the `oc get build` command and checking the new `todolist-2` build in the output:
```

Comment:
after the image, add 

or by running the `oc get build` command and checking the new `todolist-2` build in the output:
```shell
$ oc get build -n mongo-persistent
NAME         TYPE     FROM          STATUS     STARTED         DURATION
todolist-1   Docker   Git@e27a21e   Complete   8 minutes ago     1m42s
todolist-2   Docker   Git@8c045a2   Complete   2 minutes ago     1m37s
```

Reply:
changed